pkg/templates/charts/toggle/console-mce/templates/console-quickstart.yaml

```diff
@@ -0,0 +1,169 @@
+        the status of the `acm` plug-in is also enabled.
+
+        1. If the applicable plug-ins are disabled, complete the following steps for each plug-in:
+          
+            1. Click on the **Disabled** status.
+         
+            1. Select **Enable** from the dialog box.
+         
+            1. Click on **Save**.  
+
+        1. If you enabled any plug-ins, after a few moments, an alert with the text “Web console update is
+        available” appears. Click the **Refresh web console** link to load the web
+        console with the plug-ins enabled.
+      review:
+        failedTaskHelp: This task is incomplete. Try the task again.
+        instructions: |-
+          Verify that the console plug-in was enabled successfully:
+
+            - In the navigation menu, is a cluster switcher present with the options **All Clusters** and **local-cluster**? The cluster switcher should appear before the [perspective switcher]{{ "{{" }}highlight qs-perspective-switcher{{ "}}" }}.
+    - title: Understand feature gates  
+      description: |-
+        The remaining steps require you to use feature gates to activate the `TechPreviewNoUpgrade` feature set.
+
+        **Warning**: Using the `TechPreviewNoUpgrade` feature set enables Technology Preview features 
+            that are not part of the default features. Enabling this feature set 
+            cannot be undone and prevents upgrades. **This feature set is NOT 
+            recommended on production clusters**.
+
+        For more information, see [Enabling OpenShift Container Platform features using FeatureGates](https://docs.openshift.com/container-platform/4.10/nodes/clusters/nodes-cluster-enabling-features.html).
+      review:
+        instructions: |
+          Do you acknowledge that enabling the feature gate is irreversible and not recommended for production clusters?
+        failedTaskHelp: Do not proceed with the remaining steps.
+    - title: Enable feature gate 
+      description: |-
+        1. From Administrator [perspective]{{ "{{" }}highlight qs-perspective-switcher{{ "}}" }} of the OpenShift Container Platform web console, navigate to the cluster settings by clicking **[Administration]{{ "{{" }}highlight qs-nav-administration{{ "}}" }} > Cluster settings**
+
+        1. Click **FeatureGate** from the *Configuration* tab.
+
+        1. On the *YAML* tab, edit the cluster instance to add the following specific feature sets:
+
+            ```
+            spec:
+              featureSet: TechPreviewNoUpgrade
+            ```
+
+        1. Click **Save**.
+      review:
+        failedTaskHelp: This task is incomplete. Try the task again.
+        instructions: |
+          Verify that the feature gate was enabled successfully:
+
+            - In the navigation menu, click **Administration > Cluster settings**. Do you see a warning stating that the cluster cannot be updated because of the `TechPreviewNoUpgrade` feature set?
```

Comment:
i'm not sure about asking the question like this. @swopebe do you know of any reason why a question shouldn't used in the Quick Start Guide. I didn't see anything in the PatternFly guide

Instead of asking a question, I would make it similar to a verification step.

Reply:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/42188127/164555194-a188356e-1ee5-4469-bfb1-1691fdf84b06.png">

Reply:
I recommend that you write the following:

Verify that the feature gate is enabled successfully.

Reply:
Updated to "is enabled:" here https://github.com/stolostron/backplane-operator/pull/151/commits/bddaee43200acfe78abea025b94b21c7a2a3c724